diff --git js/private/BUILD.bazel js/private/BUILD.bazel
index 2218b48d..c898dfaa 100644
--- js/private/BUILD.bazel
+++ js/private/BUILD.bazel
@@ -25,7 +25,7 @@ config_setting(

 config_setting(
     name = "experimental_allow_unresolved_symlinks",
-    values = {"experimental_allow_unresolved_symlinks": "true"},
+    values = {"define": "unused=unused"},
     visibility = ["//visibility:public"],
 )

